README.md

```diff
@@ -1,2 +1,30 @@
 # liveDVR
 DVR implementation built on top of live streams
+
+# Installation #
+
+## Pre-requisites ##
+
+* Make sure node is installed
```

Comment:
Indeed. Also, it should be properly declared in package.json, for instance:
```
"engines" : { "node" : ">=6.10.2" },
```
See an example in https://github.com/kaltura/pub-sub-server/blob/Lynx-12.20.0/package.json#L5

In addition, see my comment on NVM, why is it needed?